src/code/templates/clutch-game.js

```diff
@@ -271,4 +271,11 @@ export default class ClutchGame extends Component {
     return 1;
   }
 
+  static shouldShowMovesCounter = function(challengeType) {
+    if (challengeType === "submit-parents") {
```

Comment:
Knowing that test-cross is coming, would be good to add that to the list, unless there's a way we can we pick this up from an authoring flag?